providers/task.rb

```diff
@@ -182,6 +186,46 @@ def task_need_update?
     @current_resource.user != @new_resource.user
 end
 
+def set_cwd(folder)
+  Chef::Log.debug 'looking for existing tasks'
```

Comment:
OK really nitpicking.. capitalize the "l" in "looking" to be consistent with the other debug messages